README.md

```diff
@@ -9,6 +9,19 @@ If you want to use factory_bot with Rails, see
 
 _[Interested in the project name?](NAME.md)._
 
+
+### Transitioning from factory\_girl?
+
+Transitioning is a two step process:
+
+1) Replace `factory_girl` with `factory_bot` in your Gemfile and `bundle install`
+
+2) Rename all `FactoryGirl` to `FactoryBot` and `factory_girl` to `factory_bot` in your codebase.
+
+This command will perform a `FactoryGirl` to `FactoryBot` rename within your `spec/` directory:
+
+```ag -0l FactoryGirl spec | xargs -0 sed -i 's/FactoryGirl/FactoryBot/g'```
```

Comment:
Since there are some people who don't have Silver Searcher, maybe we should use a common tool like `grep`. Alternatively, we can say something like "If you have a search tool like Silver Searcher, this command will...`.